<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graficador ECG</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(219, 219, 219)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="signal_plot_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>771</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="serial_device_comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(186, 186, 186);
color: rgb(0, 0, 0)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serial_connect_pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Connectar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Graficador ECG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Puerto Serie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_pushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Pausar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Intervalo de</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="interval_doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0)</string>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>410</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>visualizaci√≥n (s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="interval_ok_pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
